index.html

```diff
@@ -5,15 +5,70 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="stylesheet" href="../css/normalize.css">
+    <link rel="stylesheet" href="../css/style.css">
+    <link rel="shortcut icon" type="../image/x-icon"  href="favicon.ico">
 </head>
+<!-- </head> -->
 <body>
+    <header>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+        <nav>
+        <img class="logo" src="../img/karma-logo.svg" alt="Meet Karma Logo">
```

Comment:
The indentation isn't quite right here which makes it more difficult to follow the structure of the HTML. Did you install an extension to auto-fix indentation on save?

Reply:
I did, but for some reason, it is not working.

Reply:
Ok, looks like Formatter is working only for HTML, but not for CSS.